autogpt/memory/postgresml.py

```diff
@@ -0,0 +1,134 @@
+import psycopg2
+from psycopg2 import sql
+from typing import Any
+from autogpt.llm_utils import create_embedding_with_ada
```

Comment:
getting an error when trying to run your branch, doesn't look like this `create_embedding_with_ada` function exists.

Reply:
think you want `get_ada_embedding`